saleor/dashboard/shipping/forms.py

```diff
@@ -16,3 +22,10 @@ class Meta:
         model = ShippingMethodCountry
         exclude = []
         widgets = {'shipping_method': forms.widgets.HiddenInput()}
+        labels = {
+            'country_code': pgettext_lazy(
+                'Country code', 'Country code'),
```

Comment:
Do we really expect a code here? Is this not a list of countries you select from?